internal/domain/power-cap.go

```diff
@@ -659,13 +660,23 @@ func doPowerCapTask(taskID uuid.UUID) {
 		logger.Log.WithFields(logrus.Fields{"ERROR": err}).Error("Error storing power capping task")
 	}
 
+	// Repeated and frequent power caps to the same BMCs is not
+	// common so we use the default TRS configuration provided by the
+	// default BaseTRSTask task prototype.  It may be beneficial to
+	// consider sharing the PCS TRS client in the future as power capping
+	// generally shares the same set of BMC targets we want to talk to.
+	//
+	// SPC does initiatefrequent repeated http requests to BMCs so when
+	// it is added to CSM for formal release, we should consider adding
+	// a new client or sharing (and enlarging) the PCS TRS client.
+
 	// Create TRS task list
 	trsTaskMap := make(map[uuid.UUID]model.PowerCapOperation)
 	trsTaskList := (*GLOB.RFTloc).CreateTaskList(GLOB.BaseTRSTask, len(goodOps))
 	trsTaskIdx := 0
 	for _, op := range goodOps {
 		trsTaskMap[trsTaskList[trsTaskIdx].GetID()] = op
-		trsTaskList[trsTaskIdx].RetryPolicy.Retries = 3
+		trsTaskList[trsTaskIdx].CPolicy.Retry.Retries = 3
```

Comment:
Should this be set to the retries variable sent in?

Reply:
The global variable PCS_STATUS_HTTP_RETRIES is for status requests.  At some point we may also want one for power caps and transitions.